bom/pom.xml

```diff
@@ -400,7 +400,7 @@ THE SOFTWARE.
       <dependency>
         <groupId>jaxen</groupId>
         <artifactId>jaxen</artifactId>
-        <version>1.2.0</version>
+        <version>1.2.0-atlassian-2</version>
```

Comment:
Since this isn't an upgrade but rather a switch to a forked version of this dependency, I think we'd need a justification for why we want to switch to the fork. Probably this should be reverted.